### DISCUSSION TITLE
Support SAML login protocol

### DISCUSSION THREAD

**Discussion by emfhal** [langfuse/langfuse#2352] [Ideas]
SAML offers a more robust and secure framework for managing user access compared to a generic SSO approach

**Comment by marcklingen** [comment]
Agree that SAML is relevant, currently looking into various solutions to bridge NextAuth/Authjs and SAML. Let's keep the conversation open here, very much open for contributions on this

Potential solutions
- https://github.com/ssoready/ssoready
- https://next-auth.js.org/providers/boxyhq-saml
